### DISCUSSION TITLE
Prompt Cataloguing and prompt github

### DISCUSSION THREAD

**Discussion by kunal2002** [langfuse/langfuse#2573] [Ideas]
### Describe the feature or potential improvement

A feature that can be used to manage prompts based on certain metrics, such as the scores traced in Langfuse. The prompt version with the best metrics can be automatically pushed into production to facilitate a CI/CD pipeline so that manual prompt labeling can be automated.

There can also be a feature that helps to perform "pull", "push" , and "clone" prompts just like GitHub facilitates for repos, same can be done for different prompt templates to get the most out of each prompt template and it's specific version.

**Comment by marcklingen** [comment]
Agree, it'd be cool to expose the prompt metrics via api in order to enable these workflows

https://langfuse.com/changelog/2024-05-16-prompts-version-insights
